### PR TITLE
add support for sparse volumes

### PR DESCRIPTION
**Pull request by Area128**
##### SUMMARY

Allow the creation of sparse volumes (zfs create -s ...)

##### ISSUE TYPE

- Feature Pull Request

##### COMPONENT NAME

zfs

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/storage/zfs/zfs.py [hunk 1]
```diff
@@ -158,6 +166,9 @@ def create(self):
                     cmd += ['-V', value]
                 elif prop == 'volblocksize':
                     cmd += ['-b', value]
+                elif prop == 'sparse':
+                    if value == 'on':
```
Comment (mator) [inline lib/ansible/modules/storage/zfs/zfs.py]: ``

Comment (mator) [inline lib/ansible/modules/storage/zfs/zfs.py:170]: probably better to use:
```
if check_type_bool(value):
```

or just ignore (resolve) this review/comment. Thanks.

**Review by mator — COMMENTED** [review]
could use check_type_bool() function instead of a plain check for value, which could be true/false, on/off, 1/0 (including uppercase first character)...

File: lib/ansible/modules/storage/zfs/zfs.py (hunk 1 above)
Comment (Area128) [reply lib/ansible/modules/storage/zfs/zfs.py:170]: Thanks, done.

**Pull request closed without merging**